onnx/defs/loss/defs.cc

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) ONNX Project Contributors.
+// Licensed under the MIT license.
+
+#include "onnx/defs/schema.h"
+namespace ONNX_NAMESPACE {
+const char* reduction_doc =
+    "Type of reduction to apply to loss: none, sum, mean(default). "
+    "'none': no reduction will be applied, "
+    "'sum': the output will be summed. "
+    "'mean': the sum of the output will be divided by the number of "
+    "elements in the output.";
+
+static const char* MSE_ver10_doc = R"DOC(Loss function that measures the
```

Comment:
Please add equations to define this opertor's behavior. I feel using equation is the only way to eliminate uncertainty in ONNX (a mathematical standard). Notice that L2-norm is only formally defined for `vector` (aka 1-D tensor) but it looks like this operator accepts general tensors.